lib/Services/ImageDimensionMapper.php

```diff
@@ -14,12 +14,13 @@
 
 use OCP\AppFramework\Db\Mapper;
 use OCP\IDb;
+use OCP\IDBConnection;
 
 class ImageDimensionMapper extends Mapper {
 	/**
-	 * @param IDb $database
+	 * @param IDBConnection $database
 	 */
-	public function __construct(IDb $database) {
+	public function __construct(IDBConnection $database) {
 		parent::__construct($database, 'mediametadata_image_size', '\OCA\MediaMetadata\Services\ImageDimension');
```

Comment:
Please use Dependency Injection here. There is an explanation as to why in the official documentation.